mmv1/products/dataplex/Datascan.yaml

```diff
@@ -250,10 +250,27 @@ properties:
         name: 'samplingPercent'
         description: |
           The percentage of the records to be selected from the dataset for DataScan.
+          Value can range between 0.0 and 100.0 with up to 3 significant decimal digits.
+          Sampling is not applied if `sampling_percent` is not specified, 0 or 100.
       - !ruby/object:Api::Type::String
         name: 'rowFilter'
         description: |
           A filter applied to all rows in a single DataScan job. The filter needs to be a valid SQL expression for a WHERE clause in BigQuery standard SQL syntax. Example: col1 >= 0 AND col2 < 10
+      - !ruby/object:Api::Type::NestedObject
+        name: 'postScanActions'
+        description: |
+          Actions to take upon job completion.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'bigqueryExport'
+            description: |
+              If set, results will be exported to the provided BigQuery table.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'resultsTable'
+                description: |
+                  The BigQuery table to export DataQualityScan results to.
+                  Format://bigquery.googleapis.com/projects/PROJECT_ID/datasets/DATASET_ID/tables/TABLE_ID
```

Comment:
Are you able to supply a test for this field? If not that is okay I just want to ensure we are conscious in it's exclusion.

Reply:
This one requires an input table path for which the scan creator should have permission, would be tricky to set up. We can exclude it from our tests for now.

Reply:
Does the Datascan Export API create datasets if not present either? If so, we can simply specify a BQ table path with the Project being interpolated, and the test-runner will be able to create it.

Something like  project/"<%= ctx[:test_env_vars]['project_name'] %>"/dataset/testDataScan/table/testTable
----

We could also create a BQ Table via Terraform itself and reference that as part of the test setup.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/blob/051ffe1f2feed35899fbb8d0ac8c1c510f238ee9/mmv1/templates/terraform/examples/dataplex_asset_primary.tf.erb 

This is an example of a Dataplex Asset setting up other resources it needs before itself, and then referencing them.

Reply:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table_iam#google_bigquery_table_iam_member. --> We can use this to grant the Dataplex P4SA write permissions to the Table.

Reply:
Discussed with @saurabh-net offline as well to add more test cases eventually for providing more coverage.

Reply:
Looks like this proved harder than expected; agree on adding tests for this later.